project.clj

```diff
@@ -96,6 +98,15 @@
   :lein-release {:scm        :git
                  :deploy-via :lein-deploy}
 
+  :uberjar-name "puppetdb-release.jar"
+  :lein-ezbake {:vars {:user "puppetdb"
+                       :group "puppetdb"
+                       :build-type "foss"
```

Comment:
I think I like approach #2 ... keeps it altogether I guess.

So to let you know, pe-puppetdb doesn't 'work' today, we have another ticket tracking the effort to get that working: https://tickets.puppetlabs.com/browse/PDB-1032 so we could punt this until later, as long as we think we have a way that won't dramatically change the lein-ezbake plugin API too much, and its in our vision this is probably okay for now.